physx.js

```diff
@@ -612,6 +616,75 @@ const physxWorker = (() => {
       } : null;
     }
   };
+  w.groundRaycastPhysics = (physics, p, q) => {

```

Comment:
I do not think we should be doing this to detect groundedness. Groundedness should be based on support restitution.